content/en/synthetics/guide/app-that-requires-login.md

```diff
@@ -43,11 +43,15 @@ To record your steps without logging out of your application, use the recorder's
 
 Opening a pop-up in incognito mode allows you to start your test's recording from the start URL set in your test configuration with a session completely isolated from your own browser's main session and user data. The freshly opened incognito pop-up ignores all your previous browser history including cookies and local data. You are automatically logged out from your account and can start recording your login steps as if you were visiting your website for the first time.
 
-**Note:** Use [the subtest feature][2] to group your login steps into a single subtest that you can then reuse across any other browser tests that require a login.
+Create [subtests][2] to group your login steps into a single subtest that you can then reuse across any other browser tests that require a login.
 
 ### SSO login
 
-If your website uses SSO for login, input your application's URL as the starting URL of your browser test. The test performs the required redirections as part of the first default **Navigate to URL** step.
+If your website uses single sign-on (SSO) for login, input your application's SSO portal login URL as the starting URL of your browser test. Create a browser test that includes recorded steps for entering your login credentials. The test performs the required redirections as part of the first default **Navigate to URL** step. 
+
+Create a second browser test that includes a **Step 1** that is a subtest, and select the first browser test you created. Then, have it navigate to the SSO portal login link for the application you want to authenticate with.
+
+Create a third browser test on the endpoint you want to test and include a **Step 1** that is a subtest, and select the second browser test you created. Then, the subsequent tests can test additional functionality in your application.
```

Comment:
why are we recommending the user to create 3 tests to login rather than the simpliest approach and stick with 1 browser test